packages/sanity/src/core/preview/components/SanityDefaultPreview.tsx

```diff
@@ -50,6 +50,13 @@ export function SanityDefaultPreview(props: SanityDefaultPreviewProps): ReactEle
       dimensions: {width?: number; height?: number; fit: ImageUrlFitMode; dpr?: number}
     }) => {
       const {dimensions} = options
+      let imageSource = mediaProp
+
+      if (isString(imageSource) && isSanityImageUrl(imageSource)) {
```

Comment:
Can I have some comments here the same way you've done at the bottom please? 🙏

Reply:
✅